test/parallel/test-buffer-concat.js

```diff
@@ -98,3 +99,15 @@ assert.deepStrictEqual(
 assert.deepStrictEqual(Buffer.concat([new Uint8Array([0x41, 0x42]),
                                       new Uint8Array([0x43, 0x44])]),
                        Buffer.from('ABCD'));
+
+// Ref: https://github.com/nodejs/node/issues/55422#issue-2594486812
```

Comment:
Nit: please consider expanding the comment a bit here with a short summary of what is being tested for so folks don't have to necessarily follow the link to get the gist.

Reply:
I agree that it is better to have the description of the test inlined. 

However I was under the impression that the policy was to use refs?

See https://github.com/nodejs/node/pull/55492#discussion_r1815305213 where I actually removed the explanation at the request of another maintainer.